thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/detection/algorithm/stage/AnomalyDetectionStageWrapper.java

```diff
@@ -67,4 +119,86 @@ public DetectionPipelineResult run() {
   private AnomalyDetectionStage loadAnomalyDetectorStage(String className) throws Exception {
     return (AnomalyDetectionStage) Class.forName(className).newInstance();
   }
+
+  List<Interval> getMonitoringWindows() {
```

Comment:
I'd prefer passing startTime and endTime as parameters. We can convert this into a utility in the future.

Reply:
maybe not now cause this function requires passing multiple parameters from outside and intended to use only here.